.node-version

```diff
@@ -0,0 +1 @@
+9.10.0
```

Comment:
Did you mean to add this file?

Reply:
i can remove it, it's for autoswapping of node versions in [node-env](https://github.com/nodenv/nodenv) nvm alternative

Reply:
@jennyscript removed